src/mbgl/renderer/circle_bucket.hpp

```diff
@@ -28,10 +27,14 @@ class CircleBucket : public Bucket {
     void drawCircles(CircleShader&, gl::Context&);
 
 private:
-    CircleVertexBuffer vertexBuffer_;
-    TriangleElementsBuffer elementsBuffer_;
+    std::vector<CircleVertex> vertexes;
+    std::vector<gl::Triangle<uint16_t>> indexes;
```

Comment:
Ah, this is an OpenGL ES limitation and I was looking at generic OpenGL documentation for `glDrawElements`. OpenGL ES also supports `uint8_t` indexes, but I don't think we'll ever use those. So I'll deparameterize this.